app-config.yaml

```diff
@@ -5,6 +5,11 @@ app:
   support:
     url: https://github.com/backstage/backstage/issues # Used by common ErrorPage
     items: # Used by common SupportButton component
+      - title: TechDocs Support
+        icon: docs
+        links:
+          - url: https://discord.gg/MUpMjP2
+            title: '#docs-like-code'
```

Comment:
So the only issue with this I believe is this is global - this isn't JUST for TechDocs, it'll now add this link across the entire Backstage instance for every single support button on all components and all plugins.  Was that what was intended?

I have a feeling this should instead be done when/if we can extend the support component to inherit component- or plugin-specific support elements?

Reply:
Yeah add it globally is the only option right now, so either that or leave it out of this PR.

Reply:
@adamdmharvey I'll leave the support config for TechDocs out of this PR fyi!